deps/ledger-zxlib/include/zxformat.h

```diff
@@ -183,21 +183,31 @@ __Z_INLINE uint8_t fpstr_to_str(char *out, uint16_t outLen, const char *number,
         return 1;
     }
 
+
     if (digits <= decimals) {
-        // First part
-        strcpy(out, "0.");
+        if (outLen <= 2 + decimals)
+        {
+          snprintf(out, outLen, "ERR");
+          return 1;
+        }
+        strlcpy(out, "0.", 2);
```

Comment:
Size value for `strlcpy` must be the size of the destination buffer, and nothing else.
This pattern is dangerous (and bugged: it will copy "`0\0`" in `out`, not "`0.`".